instaloader/structures.py

```diff
@@ -49,6 +49,7 @@ class PostComment(NamedTuple):
     text: str
     owner: 'Profile'
     likes_count: int
+    likes: Iterator['Profile']
```

Comment:
Docstring for likes is missing.

The docstrings for PostComment are described in currently line 56 - 58 of this file.